docker/Dockerfile

```diff
@@ -14,19 +16,12 @@ RUN apt-get update && \
 		ffmpeg \
 		git \
 		graphicsmagick-imagemagick-compat \
+		potrace \
 		libboost-all-dev \
 		libeigen3-dev \
 		libopencv-dev \
 		libpng-dev \
 		libpng++-dev \
-		potrace \
-	&& apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN mkdir -p $APP_HOME/
-
-RUN apt-get update && \
-	apt-get install -qq -y \
```

Comment:
got rid of two-step `apt-get install` for image efficiency. it was left in while I was still working through getting all the right libraries